cs5530/UserService.java

```diff
@@ -0,0 +1,50 @@
+package cs5530;
+
+import java.lang.*;
+import java.sql.*;
+import java.io.*;
+//import javax.servlet.http.*;
+
+public class UserService {
+    // userType 0 is UberUser
```

Comment:
Is there a reason for userType to be an int? Could we make it a string that's equal to the Table name, and just use that rather than having that if/else tree in createUser?